src/test/e2e/31_checksum_and_signature_test.go

```diff
@@ -36,7 +36,7 @@ func TestChecksumAndSignature(t *testing.T) {
 	// Test that we get an error when trying to deploy a package without providing the public key
 	stdOut, stdErr, err = e2e.Zarf(t, "package", "deploy", pkgName, "--confirm")
 	require.Error(t, err, stdOut, stdErr)
-	require.Contains(t, stdErr, "failed to deploy package: unable to load the package: package is signed but no key was provided - add a key with the --key flag or use the --skip-signature-validation flag and run the command again")
+	require.Contains(t, stdErr, "unable to load package: package is signed but no key was provided")
```

Comment:
We'll have to decide how descriptive we want error messages to be on deploy. I would say it's reasonably intuitive to look at flags given the new message. A reasonable counter would be that Zarf is constructed in such a way that it can be deployed by an inexperienced person, though the typical deployer is likely more often a pipeline nowadays.

Do we consider the original message overkill or should it be included for more clarity. I lean to keep the error message smaller, but I can see it either way.

Reply:
I'm leaning towards the smaller error message approach that you mentioned and that's in the current branch. 

For this particular test assert, checking that an error occurred is plenty. The test case and inputs are clear and we get no additional information or test coverage by requiring a error string.

Reply:
IMO for e2e tests it's worth considering checking stderr for a specific string. There is a fair amount that could go wrong during deploy, so this guards against the test failing for some other reason